src/wms/src/main/java/org/geoserver/wms/capabilities/CapabilitiesXmlReader.java

```diff
@@ -124,6 +124,8 @@ public void startElement(
                         request.setVersion(version);
                     } else if (atts.getLocalName(i).equals("updateSequence")) {
                         request.setUpdateSequence(atts.getValue(i));
+                    } else if (atts.getLocalName(i).equals("noRootLayer")) {
```

Comment:
Is this worth it for what is a vendor option override? What is the goal here force show or force hide the root layer on a client by client basis?

Reply:
Support for POST is not needed by customer, if you prefer I can remove it

Reply:
It would not be the first option that is available in KVP only, however users have complained in the past about this asymmetry (e.g. viewParams in WFS GET but not WFS POST, however for example I don't believe you can pass viewParams in WMS GetMap POST). I'm open to both approaches (documenting, or removing).